include/LightGBM/config.h

```diff
@@ -744,6 +744,14 @@ struct Config {
   // desc = `NDCG <https://en.wikipedia.org/wiki/Discounted_cumulative_gain#Normalized_DCG>`__ and `MAP <https://makarandtapaswi.wordpress.com/2012/07/02/intuition-behind-average-precision-and-map/>`__ evaluation positions, separated by ``,``
   std::vector<int> eval_at;
 
+  // check = >0
+  // desc = used only with ``multi-error`` metric
+  // descl2 = threshold for top-k multi-error metric
+  // descl2 = the error on each sample is ``0`` if the true class is among the top ``multi_error_top_k`` predictions, and ``1`` otherwise
+  // descl2 = more precisely, the error on a sample is ``0`` if there are at least ``num_classes - multi_error_top_k`` predictions strictly less than the prediction on the true class
```

Comment:
I'm sorry, I had to be more clear! `descl2` is only for this one clarification line, which starts from `more precisely ...` and was in brackets. All other lines should remain marked as `desc`.